aiohttp/web_urldispatcher.py

```diff
@@ -91,53 +84,113 @@ def _append_query(url, query):
             return url
 
 
-class PlainRoute(Route):
+class ViewableRoute(Route):
+
+    def __init__(self, name, *, expect_handler=None):
+        super().__init__(name, expect_handler=expect_handler)
+        self._views = []
```

Comment:
I believe that list is temporary solution as on practice here should be tree-like thing because Views may serve various content type depending on Accept-* headers.